2016-02-19 16:30:42,821:821.573019028 - ubuntu_sso.utils.translation - WARNING - Translations not found, using null translator.
2016-02-19 16:30:42,881:881.067991257 - ubuntu_sso.main - INFO - Setting up Ubuntu SSO service.
2016-02-19 16:30:42,897:897.732973099 - ubuntu_sso.main - INFO - Ubuntu SSO service started.
2016-02-19 16:30:42,898:898.009061813 - ubuntu_sso.account - INFO - Creating a new SSO access layer for service url u'https://login.ubuntu.com/api/1.0/'
2016-02-19 16:30:42,925:925.520896912 - ubuntu_sso.credentials - INFO - find_credentials: self.app_name dbus.String(u'Ubuntu Software Center'), result is {}? True
2016-02-19 16:30:42,925:925.843000412 - ubuntu_sso.main - INFO - 'CredentialsManagement': emitting 'CredentialsNotFound' with args (dbus.String(u'Ubuntu Software Center'),) and kwargs {}
2016-02-19 16:30:52,937:937.763929367 - ubuntu_sso.main - INFO - Shutting down, calling <function shutdown_func at 0xb6af880c>.
2016-02-19 16:39:10,654:654.614925385 - ubuntu_sso.utils.translation - WARNING - Translations not found, using null translator.
2016-02-19 16:39:11,415:415.544986725 - ubuntu_sso.main - INFO - Setting up Ubuntu SSO service.
2016-02-19 16:39:11,437:437.829017639 - ubuntu_sso.main - INFO - Ubuntu SSO service started.
2016-02-19 16:39:11,438:438.079118729 - ubuntu_sso.account - INFO - Creating a new SSO access layer for service url u'https://login.ubuntu.com/api/1.0/'
2016-02-19 16:39:11,591:591.73989296 - ubuntu_sso.credentials - INFO - find_credentials: self.app_name dbus.String(u'Ubuntu Software Center'), result is {}? True
2016-02-19 16:39:11,592:592.082977295 - ubuntu_sso.main - INFO - 'CredentialsManagement': emitting 'CredentialsNotFound' with args (dbus.String(u'Ubuntu Software Center'),) and kwargs {}
2016-02-19 16:39:21,616:616.906881332 - ubuntu_sso.main - INFO - Shutting down, calling <function shutdown_func at 0xb670a33c>.
2016-02-19 16:44:13,805:805.283069611 - ubuntu_sso.utils.translation - WARNING - Translations not found, using null translator.
2016-02-19 16:44:14,086:86.0180854797 - ubuntu_sso.main - INFO - Setting up Ubuntu SSO service.
2016-02-19 16:44:14,124:124.063014984 - ubuntu_sso.main - INFO - Ubuntu SSO service started.
2016-02-19 16:44:14,124:124.47810173 - ubuntu_sso.account - INFO - Creating a new SSO access layer for service url u'https://login.ubuntu.com/api/1.0/'
2016-02-19 16:44:14,274:274.480104446 - ubuntu_sso.credentials - INFO - find_credentials: self.app_name dbus.String(u'Ubuntu Software Center'), result is {}? True
2016-02-19 16:44:14,275:275.094985962 - ubuntu_sso.main - INFO - 'CredentialsManagement': emitting 'CredentialsNotFound' with args (dbus.String(u'Ubuntu Software Center'),) and kwargs {}
2016-02-19 16:44:24,286:286.926031113 - ubuntu_sso.main - INFO - Shutting down, calling <function shutdown_func at 0xb675d33c>.
2016-02-22 16:26:42,238:238.594055176 - ubuntu_sso.utils.translation - WARNING - Translations not found, using null translator.
2016-02-22 16:26:42,296:296.180963516 - ubuntu_sso.main - INFO - Setting up Ubuntu SSO service.
2016-02-22 16:26:42,302:302.774906158 - ubuntu_sso.main - INFO - Ubuntu SSO service started.
2016-02-22 16:26:42,302:302.985906601 - ubuntu_sso.account - INFO - Creating a new SSO access layer for service url u'https://login.ubuntu.com/api/1.0/'
2016-02-22 16:26:42,311:311.758041382 - ubuntu_sso.credentials - INFO - find_credentials: self.app_name dbus.String(u'Ubuntu One'), result is {}? True
2016-02-22 16:26:42,311:311.918973923 - ubuntu_sso.main - INFO - 'CredentialsManagement': emitting 'CredentialsNotFound' with args (dbus.String(u'Ubuntu One'),) and kwargs {}
2016-02-22 16:26:52,321:321.032047272 - ubuntu_sso.main - INFO - Shutting down, calling <function shutdown_func at 0xb67532cc>.
2016-02-22 16:31:42,474:474.729061127 - ubuntu_sso.utils.translation - WARNING - Translations not found, using null translator.
2016-02-22 16:31:42,490:490.529060364 - ubuntu_sso.main - INFO - Setting up Ubuntu SSO service.
2016-02-22 16:31:42,494:494.955062866 - ubuntu_sso.main - INFO - Ubuntu SSO service started.
2016-02-22 16:31:42,495:495.085954666 - ubuntu_sso.account - INFO - Creating a new SSO access layer for service url u'https://login.ubuntu.com/api/1.0/'
2016-02-22 16:31:42,503:503.257036209 - ubuntu_sso.credentials - INFO - find_credentials: self.app_name dbus.String(u'Ubuntu One'), result is {}? True
2016-02-22 16:31:42,503:503.46493721 - ubuntu_sso.main - INFO - 'CredentialsManagement': emitting 'CredentialsNotFound' with args (dbus.String(u'Ubuntu One'),) and kwargs {}
2016-02-22 16:31:52,507:507.112026215 - ubuntu_sso.main - INFO - Shutting down, calling <function shutdown_func at 0xb66f12cc>.
